packages/core/src/binary-data/utils.ts

```diff
@@ -58,8 +58,18 @@ export const FileLocation = {
 	 * Create a location for a binary file at a custom path,
 	 * e.g. ["chat-hub", "sessions", "abc", "messages", "def"] -> "chat-hub/sessions/abc/messages/def"
 	 */
-	ofCustom: (pathSegments: string[]): BinaryData.FileLocation => ({
+	ofCustom: ({
```

Comment:
I wonder if this was clearer if we had `ofChatMessageAttachment` that took `({ pathSegments, messageId })`?